testsuite/tests/native-debugger/gdb-script

```diff
@@ -0,0 +1,15 @@
+break *(&caml_start_program+0)
```

Comment:
Curious detail here with `gdb` when setting a breakpoint using just `br caml_start_program` GDB was setting the actual breakpoint at some offset from that symbol in the assembly. On AMD64 it was directly on that symbol but for ARM64 it is at `caml_start_program+4` and for RISC-V it ends up at `caml_start_program+40`.  So I have used an offset here. If you remove it then the GDB session on RISC-V is incorrect as we re-enter via [`caml_callback_asm`](https://github.com/ocaml/ocaml/blob/trunk/runtime/riscv.S#L882-L906) and [`jump_to_caml` label]( https://github.com/ocaml/ocaml/blob/trunk/runtime/riscv.S#L623) which incorrectly hits that breakpoint.